addons/l10n_uy/tests/test_manual.py

```diff
@@ -0,0 +1,90 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.tests.common import Form, tagged
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged('post_install_l10n', '-at_install', 'post_install')
+class TestManual(AccountTestInvoicingCommon):
```

Comment:
It does not seem that useful to have tests just for 2 validations...

Reply:
Yes indeed, there is not a lot to test at this point. We added the unit test because it was requested by Odoo team. We can remove it if you like @jco-odoo  please let me know

Reply:
Please remove.